docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,392 @@
+   spec:
+     output:
+       custom: |
+         Name               http
+         Host               https://myhost/logs
+         Tls                On
+   
+   ```
+
+   An output is a data destination that can be configured by a [Fluent Bit output](https://docs.fluentbit.io/manual/pipeline/outputs) of the relevant type. Here, the used output type is `http`, which is configured with the relevant parameters.
+
+2. To create the instance, apply the resource file in your cluster.
+
+   ```bash
+   kubectl apply -f path/to/my-log-pipeline.yaml
+   ```
+
+3. Check that the status of the pipeline in your cluster is `Ready`:
+
+   ```bash
+   kubectl get logpipeline
+   NAME              STATUS    AGE
+   http-backend      Ready     44s
+   ```
+
+### Step 2: Add filters
+
+To enrich logs with attributes or drop whole lines, add filters to the existing pipeline:
+
+```yaml
+kind: LogPipeline
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: http-backend
+spec:
+  filters:
+    - custom: |
+        Name    grep
+        Regex   $kubernetes['labels']['app'] my-deployment
+    - custom: |
+        Name    grep
+        Exclude $kubernetes['namespace_name'] kyma-system|kube-system|kyma-integration|istio-system
+    - custom: |
+        Name    record_modifier
+        Record  cluster_identifier ${KUBERNETES_SERVICE_HOST}
+  output:
+    ...
+```
+
+In the example, three filters are added, which are executed in sequence. Different types of [Fluent Bit filter](https://docs.fluentbit.io/manual/concepts/data-pipeline/filter) are supported. The example uses the [grep](https://docs.fluentbit.io/manual/pipeline/filters/grep) and the [record_modifier](https://docs.fluentbit.io/manual/pipeline/filters/record-modifier) filter.
+
+- The first filter keeps all log records that have the `kubernetes.labels.app` attribute set with value `my-deployment`; all other logs are discarded. The `kubernetes` attribute is available for every log record. See [here](#kubernetes-filter-metadata) for more details.
+- The second filter drops all log records fulfilling the given rule. Here, typical Namespaces are dropped based on the `kubernetes` attribute.
+- A log record is modified by adding a new attribute. Here, a constant attribute is added to every log record for recording the actual cluster node name at the record for later filtering in the backend system. As value, a placeholder is used referring to a Kubernetes-specific environment variable.
+
+### Step 3: Add authentication details from secrets
+
+Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in secrets, the `LogPipeline` supports the reference of Secrets. The key-value entries of the Secrets can be mapped to environment variables of the Fluent Bit Pods, and with that, are available for usage in [placeholder expressions](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/variables).
```

Comment:
Is the first "secret" here unrelated to Kubernetes resources and that's why it's not [capitalized](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#capitalized-terms)?

Reply:
```suggestion
Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in Secrets, the `LogPipeline` supports the reference of Secrets. The key-value entries of the Secrets can be mapped to environment variables of the Fluent Bit Pods, and with that, are available for usage in [placeholder expressions](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/variables).
```